src/services/feedbackService.js

```diff
@@ -0,0 +1,49 @@
+import models from '../database/models';
+
+const { Trips, Feedback } = models;
+
+const checkWhoFeedback = async (userId, accomodationId) => {
+  try {
+    const trip = await Trips.findOne({ where: { userId, accomodationId } });
+    const { dateOfTravel, dateOfReturn } = trip.dataValues;
+    const travelDays =
+      Math.abs(dateOfReturn - dateOfTravel) / (1000 * 3600 * 24);
```

Comment:
Don't use the dateOfTravel to compute the time spent in accommodation, use updatedAt. If the the requester will spend more than 1 day in the accommodation, this method will always return true